setup.py

```diff
@@ -37,7 +37,9 @@
         'scipy~=1.2',
         'dash-daq~=0.1',
         'webviz-config>=0.0.4',
-        'webviz-subsurface-components>=0.0.3'
+        'webviz-subsurface-components>=0.0.3',
+        'pillow',
+        'xtgeo==2.1.0.b17'
```

Comment:
Is an official release of `xtgeo` with required functionality soon to come? cc: @jcrivenaes

Reply:
Yes I can make 2.1.0 on pypi quite soon.